app/scripts/components/facets/facets.controllers.ts

```diff
@@ -98,6 +99,10 @@ module ngApp.components.facets.controllers {
       this.FacetService.addTerm(facet, term);
     }
 
+    addMissing(facet: string): void {
+      this.FacetService.addTerm(facet, 'MISSING');
```

Comment:
not clear on your sentence, but the line I suggested sends 'op': 'is'. 'or' is needed in that case but the current facets display doesn't support 'or'. @shanewilson's saying to update the api to support 'in' for missing?